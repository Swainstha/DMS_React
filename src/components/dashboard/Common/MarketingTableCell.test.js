import React from 'react';
import MarketingTableCell from './MarketingTableCell';

import {
    configure,
    shallow,
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})



const setup = (props = {}) => {
    return shallow( < MarketingTableCell
     {...props}  />
    )
}

const findByTestAttr=(wrapper, val)=>{
    return wrapper.find(`[data-test='${val}']`)
}
export const dummy_data = { data:
    {id:1, name:"hello", desc:"abc", icon:"https://ff.com/df/350x350", color:'green', budget:3000, growth:"dk"}}

 it("snapshot test MarketingTableCell", () => {
        let wrapper = setup(dummy_data);
        expect(wrapper).toMatchSnapshot();
}) 