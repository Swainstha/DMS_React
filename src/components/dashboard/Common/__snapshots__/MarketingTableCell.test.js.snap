// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test MarketingTableCell 1`] = `ShallowWrapper {}`;
